app.rb

```diff
@@ -0,0 +1,137 @@
+require './Classes/music_album'
+require './Classes/list_creator'
+require './Classes/data'
+require './Classes/book'
+require './Classes/game'
+require './Classes/author'
+require './Classes/label'
+
+class App
+  def initialize
+    @books = Book.convert_to_obj(Data.read_from_file('books.json'))
+    @labels = Label.convert_to_obj(Data.read_from_file('labels.json'))
+    @music_albums = MusicAlbum.convert_to_obj(Data.read_from_file('music_albums.json'))
+    @genres = Data.read_from_file('genres.json')
+    @games = Game.convert_to_obj(Data.read_from_file('games.json'))
+    @authors = Author.convert_to_obj(Data.read_from_file('authors.json'))
+  end
+
+  def start_menu
+    puts "Please choose an option by entering a number:\n
+    0.- List all books
+    1.- Add a book
+    2.- List of games
+    3.- Add a game
+    4.- List all music albums
+    5.- Add a music album
+    6.- List all genres (e.g 'Comedy', 'Thriller')
+    7.- List all labels (e.g. 'Gift', 'New')
+    8.- List all authors (e.g. 'Stephen King')
+    9.- Exit "
+    answer = gets.chomp.to_i
+    options(answer)
+  end
+
+  def options(answer)
+    book_option(answer)
+    game_option(answer)
+    music_option(answer)
+    general_option(answer)
+  end
+
+  def book_option(answer)
+    case answer
+    when 0
+      ListCreator.new.list_all('books', @books)
+    when 1
+      puts "Creating a new book...\n\n"
+      print 'What date was it published on? (YYYY-MM-DD) '
+      publish_date = gets.chomp
+      print 'Who was the Publisher?: '
+      publisher = gets.chomp
+      print 'What state is the cover in? '
+      cover_state = gets.chomp
+      @books << Book.new(publisher, cover_state, publish_date)
+      puts 'Book created succesfully!!!'
+      print 'Would you like to add a label [Y/N] '
+      return unless answer_yes?
+
+      add_label
+    end
+  end
+
+  def answer_yes?
+    answer = gets.chomp
+    until %w[y yes n no true false].include?(answer.downcase)
+      print 'Wrong option, please enter [Y/N] '
+      answer = gets.chomp
+    end
+    %w[y yes true].include?(answer.downcase)
+  end
+
+  def add_label
+    puts "What is the label's title? "
+    title = gets.chomp
+    puts "What is the label's color? "
+    color = gets.chomp
+    @labels << Label.new(title, color)
+    puts 'Label created succesfully!!!'
+  end
```

Comment:
- [ ] I was never asked to add genre or author when creating a music album and game respectively. Kindly ask the user to add genre and author when creating a music album and game respectively. I highlighted this part of the code about adding a book and label because I was able to add a label when creating a book which means that the book is well created. Kindly follow the book and label style and implement the genre and author so that users can add them and also see them when we list all genre and author.

Reply:
done, thanks!